schedule_lessons/static/index.js

```diff
@@ -2,7 +2,25 @@ let tutorID = undefined;
 let testUrl = undefined;
 let tutor = undefined;
 
+
+
 $(document).ready(function () {
+
+    $('#defaultEntry1').timeEntry().change(function() {
+      var log = $('#log');
+      log.val(log.val() + ($('#defaultEntry').val() || 'blank') + '\n');
+    });
```

Comment:
I would rename some of the variables in this section - it is hard to understand what the variable is doing when it is called `defaultEntry1` or `log`, I dont know what this code is for and it is hard for another developer to understand so maybe try renaming them to something else? Lmk if that doesn't make sense :)